content/guide/13-advanced.md

```diff
@@ -5,33 +5,33 @@ title: Advanced
 
 ### Keyed each blocks
 
-Associating a *key* with a block allows Svelte to be smarter about how it adds and removes items to and from a list. To do so, add `@key` to the end of the opening tag for the each block, where `key` is some property that uniquely identifies each member of the list:
+Associating a *key* with a block allows Svelte to be smarter about how it adds and removes items to and from a list. To do so, add an `(expression)` that uniquely identifies each member of the list:
```

Comment:
Should indicate the (current) restrictions on what the expression can be.

Reply:
done